src/robusta/cli/main.py

```diff
@@ -126,11 +127,7 @@ def write_values_file(output_path: str, values: HelmValues):
     with open(output_path, "w") as output_file:
         yaml.safe_dump(values.dict(exclude_defaults=True), output_file, sort_keys=False)
         typer.secho(
-            f"Saved configuration to {output_path}",
-            fg="green",
-        )
-        typer.secho(
-            f"Save this file for future use. It contains your account credentials",
+            f"Saved configuration to {output_path} - save this file for future use!",
             fg="red",
```

Comment:
Why red?
I think it hints there's an error

In all other places we print red only on error

Reply:
I tried doing them both in green but this and the next message just merged together visually. I think it's good this way and it's red as it requires action on your part. It's warning you to save the file.